steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019MCRecon_KF_WithSpacing.lcsim

```diff
@@ -0,0 +1,175 @@
+            <unconstrainedV0VerticesColName>UnconstrainedV0Vertices_KF</unconstrainedV0VerticesColName>
+            <beamConV0CandidatesColName>BeamspotConstrainedV0Candidates_KF</beamConV0CandidatesColName>
+            <beamConV0VerticesColName>BeamspotConstrainedV0Vertices_KF</beamConV0VerticesColName>
+            <targetConV0CandidatesColName>TargetConstrainedV0Candidates_KF</targetConV0CandidatesColName>
+            <targetConV0VerticesColName>TargetConstrainedV0Vertices_KF</targetConV0VerticesColName>
+            <finalStateParticlesColName>FinalStateParticles_KF</finalStateParticlesColName>
+            <otherElectronsColName>OtherElectrons_KF</otherElectronsColName>
+	    <includeUnmatchedTracksInFSP>true</includeUnmatchedTracksInFSP>		       
+            <beamPositionX> 0.0  </beamPositionX>
+            <beamSigmaX> 0.3 </beamSigmaX>
+            <beamPositionY> 0.0 </beamPositionY>
+            <beamSigmaY> 0.02 </beamSigmaY>
+            <beamPositionZ> 0.0 </beamPositionZ>
+            <trackClusterTimeOffset>25</trackClusterTimeOffset>
+<!--            <maxMatchDt>40</maxMatchDt> -->
+	    <maxMatchDt>1000</maxMatchDt>
+            <useInternalVertexXYPositions>false</useInternalVertexXYPositions>
+            <minVertexChisqProb> 0.0 </minVertexChisqProb>
+            <maxVertexClusterDt>40.0</maxVertexClusterDt>
+	    <maxElectronP>7.0</maxElectronP>
+            <maxVertexP>7.0</maxVertexP>
+            <requireClustersForV0>false</requireClustersForV0>
+	    <useCorrectedClusterPositionsForMatching>false</useCorrectedClusterPositionsForMatching>
+            <applyClusterCorrections>true</applyClusterCorrections>
+            <useTrackPositionForClusterCorrection>true</useTrackPositionForClusterCorrection>  
+	    <isMC>true</isMC>
+	    <debug>false</debug>	    
+        </driver>         
+        <driver name="GBLRefitterDriver" type="org.hps.recon.tracking.gbl.GBLRefitterDriver"/>        
+        <driver name="KalmanPatRecDriver" type="org.hps.recon.tracking.kalman.KalmanPatRecDriver">
+          <!-- <doDebugPlots>true</doDebugPlots> --> 
+          <seedCompThr>0.05</seedCompThr>
+          <verbose>false</verbose>
+        </driver>
+        <driver name="TrackTruthMatching_KF" type="org.hps.analysis.MC.TrackToMCParticleRelationsDriver">
+            <trackCollectionName>KalmanFullTracks</trackCollectionName>
+            <kalmanTracks>true</kalmanTracks>
+            <debug>false</debug>
+        </driver>
+      
+        <driver name="LCIOWriter" type="org.lcsim.util.loop.LCIODriver">
+            <outputFilePath>${outputFile}.slcio</outputFilePath>
+        </driver>
+        <driver name="CleanupDriver" type="org.lcsim.recon.tracking.digitization.sisim.config.ReadoutCleanupDriver"/>
+
+	<driver name="KFOutputDriver" type="org.hps.recon.tracking.kalman.KFOutputDriver">
+          <outputPlotsFilename>${outputFile}.root</outputPlotsFilename>
```

Comment:
Are we planning to save this output? I wonder whether we should do the same for data. Might be nice to have this.

By the way, did you ever merge the KFOutputDriver into hps-java? I think it's private code you have locally only?

Reply:
I will do this in a separate pull request.